vaadin-notification.html

```diff
@@ -19,20 +19,22 @@
 
   <script>
     {
-
       /**
-       * `<vaadin-element>` is a Polymer 2 element.
+       * `<vaadin-notification>` is a Polymer 2 element providing an accessible and customizable notifications (toasts).
```

Comment:
Remove the “an” article: “—providing accessible and customizable notifications (toasts).”

Reply:
Done.